DHL/JSON/9df77954-633e-4a19-8e78-a2e73e34050f.json

```diff
@@ -0,0 +1,998 @@
+                                    {
+                                      "RulId": "607d8481-ba7c-4c95-ae07-8273cc077f59",
+                                      "Nm": "Rule condition to Repair Type",
+                                      "Cond": "**.Att[Id='75e0a595-ab28-4e91-9949-fab52cdf82b5'].Val='Opt1'",
+                                      "AttrMtdt": [
+                                        {
+                                          "isMan": true,
+                                          "isHidden": false
+                                        }
+                                      ]
+                                    }
+                                  ]
+                                },
+                                {
+                                  "Id": "18b014c0-2dd2-4888-850b-e52e28e24657",
+                                  "Nm": "Purpose Name - Repair Type",
+                                  "Dsc": "Purpose Name - Repair Type desc",
+                                  "AttTy": "Constant",
+                                  "DflVal": "User does not have Repair Type",
+                                  "isExcl": false,
+                                  "isMan": false,
+                                  "Rul": [
+                                    {
+                                      "RulId": "f44b3e0f-4f59-4df7-a20e-e95f296de3c3",
+                                      "Nm": "Agree to Repair type purpose Description",
+                                      "Cond": "**.Att[Id='75e0a595-ab28-4e91-9949-fab52cdf82b5'].Val='Opt2'and **.Att[Id='9a5247db-2c9e-4185-9991-45aa04a0b29d'].Val='Opt2'",
+                                      "AttrMtdt": [
+                                        {
+                                          "DflVal": "User doesn't have Repair Type"
+                                        }
+                                      ]
+                                    }
+                                  ]
+                                },
+                                {
+                                  "Id": "e8a10bae-c136-42ac-a12c-583a95fa51a0",
+                                  "Nm": "User have Repair Type",
+                                  "Dsc": "User have Repair Type Decription",
+                                  "AttTy": "Constant",
+                                  "DflVal": "User doesn't have Repair Type",
+                                  "isExcl": false,
+                                  "isMan": false,
+                                  "Rul": [
+                                    {
+                                      "RulId": "25b0b47d-b250-4054-ba33-d4d7b7947ead",
+                                      "Nm": "Not agree to Repair type purpose Description",
+                                      "Cond": "**.Att[Id='75e0a595-ab28-4e91-9949-fab52cdf82b5'].Val='Opt1'and **.Att[Id='9a5247db-2c9e-4185-9991-45aa04a0b29d'].Val='Opt1'",
```

Comment:
change like -> "**.Att[Id='75e0a595-ab28-4e91-9949-fab52cdf82b5'].Val='Opt1'and **.Att[Id='9a5247db-2c9e-4185-9991-45aa04a0b29d'].Val='Opt2'"

Reply:
Done